docs/sources/mimir/manage/tools/mimirtool.md

```diff
@@ -577,7 +577,7 @@ Additionally, you can control the response format using the `--use-chunks` flag:
 
 - `--use-chunks=true` (default): Requests chunked streaming response for better performance with large datasets
 - `--use-chunks=false`: Requests traditional sampled response format
-- `--chunk-digest`: Print chunk metadata (min time, max time, checksum) instead of decoding samples when using chunked responses. Can only be combined with `--use-chunks`
+- `--chunk-digest`: Print chunk metadata (min time, max time, SHA256 checksum) instead of decoding samples when using chunked responses. Can only be combined with `--use-chunks`
```

Comment:
```suggestion
- `--chunk-digest`: Prints chunk metadata (minimum time, maximum time, andSHA256 checksum) instead of decoding samples when using chunked responses. You can only combine this flag with `--use-chunks`.
```